### DISCUSSION TITLE
getUrl() not returning correct url

### DISCUSSION THREAD

**Discussion by vtxmg** [spatie/laravel-medialibrary#2003] [Q&A]
When I use getUrl() function, it returns http://localhost/project/storage/8/conversions/b-thumb.jpg
Here after storage ... app/public ... is missing.
How to solve it.. 
Note: I already used php artisan storage:link and restarted the server.

**Comment by githubtagh** [comment]
Hello Friend you have to change app_url in your .env file to correct url  then restart the server
for example APP_URL=http://127.0.0.1:8000 look at :8000 or your running port

**Reply by vtxmg** [reply]
@githubtagh my APP_URL = http://localhost/project
I have used laragon as my localserver so I think port is also fine
